cmd/juju/caas/remove.go

```diff
@@ -88,14 +88,18 @@ func (c *RemoveCAASCommand) Init(args []string) (err error) {
 	}
 	c.cloudName = args[0]
 	c.controllerName, err = c.ControllerNameFromArg()
-	if err != nil {
+	if err != nil && errors.Cause(err) != modelcmd.ErrNoControllersDefined {
 		return errors.Trace(err)
 	}
 	return cmd.CheckEmpty(args[1:])
 }
 
 // Run is defined on the Command interface.
 func (c *RemoveCAASCommand) Run(ctxt *cmd.Context) error {
+	if c.controllerName == "" && !c.Local {
+		ctxt.Stdout.Write(
+			[]byte("There are no controllers running.\nRemoving cluster from local cache. You will no longer be able to bootstrap on this cluster.\n"))
```

Comment:
Perhaps confirmation if it is removing locally without them explicitly saying?

Reply:
returned an error instead with a suitable message.
the use needs to type --local anyway so better to retain just that option rather than adding a --yes as well